src/main/java/com/almightyqa/TestUtils.java

```diff
@@ -0,0 +1,122 @@
+package com.almightyqa;
+
+import com.github.javafaker.Faker;
+import com.google.gson.Gson;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.util.stream.IntStream;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class TestUtils {
+
+    public static final Random RANDOM = new Random();
+    public static final Faker FAKER = new Faker();
+
+    public static final Gson GSON = new Gson();
+
+    public static int getRandomInt(int... bound) {
+        return bound.length == 0 ? RANDOM.nextInt() : RANDOM.nextInt(bound[0]);
+    }
+
+    public static long getRandomLong(long... bound) {
+        return bound.length == 0 ? RANDOM.nextInt() : RANDOM.nextLong(bound[0]);
+    }
+
+    public static long getRandomNumberOfGivenSize(int numberOfDigits) {
+        long min = 1;
+        if(numberOfDigits>19){
+            throw new IllegalArgumentException("Max number for long is 19 digits long");
+        }
+        for (int i = 1; i < numberOfDigits; i++) {
+            min *= 10;
+        }
+        return min + getRandomLong(9L * min) - 1;
+    }
+
+    public static String getRandomEmail(final String domain) {
+        return FAKER.name().firstName() +
+                Math.abs(getRandomInt()) +
+                "@" + domain + ".com";
+    }
+
+    public static String getRandomBankAccount() {
+        return String.format("%08d", getRandomInt(100000000));
+    }
+
+    public static String getRandomTaxpayerId() {
+        return 11 + getRandomBankAccount();
+    }
```

Comment:
maybe too project specific

Reply:
No it's a structure, potman has the same

Reply:
ok